web-app/js/portal/filter/Filter.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2015 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ *
+ */
+
+Ext.namespace('Portal.filter');
+
+Portal.filter.Filter = Ext.extend(Object, {
+
+    constructor: function(filterDetail) {
+
+        this.name = filterDetail.name;
+        this.displayLabel = filterDetail.label;
+        this.type = filterDetail.type;
+        this.visualised = filterDetail.visualised;
+        this.wmsStartDateName = filterDetail.wmsStartDateName;
+        this.wmsEndDateName = filterDetail.wmsEndDateName;
+    },
+
+    setValue: function(value) {
```

Comment:
I recall discussing this @dnahodil.  I think getters are just fluff here.  We could put them back in if and when needed?